junit-common/src/main/java/org/jboss/weld/junit/common/ContextImpl.java

```diff
@@ -27,7 +27,7 @@
 
 /**
  *
- * @author Martin Kouba
```

Comment:
I'm still the original author ;-)

Reply:
Oops, sorry, my IDE probably thought it no longer valid with those copies and moves :)